pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -205,6 +205,26 @@ public static Profile addParticipantProfileFromFile(String esIndex, String fileN
         }
     }
 
+    public static void addParticipantProfileFromTemplate(String esIndex, String ddpParticipantId, String shortId, String firstName,
```

Comment:
Given my prior comments, I am not sure why you kept this method.
Can you explain?

Reply:
Of course!
I understand your concerns regarding the method in question. My interpretation was that the primary issue was with the initial implementation of the method and its inability to perform as intended. With your input, I've been able to revise the method to address these concerns.

I believe this method remains valuable for dynamically creating profiles with specific values for testing, which would mean having a more flexible and maintainable approach than creating multiple JSON files for each test scenario. This method allows us to easily adjust test parameters and conditions without the overhead of managing numerous files.

However, I'm open to further discussions or suggestions if you believe there's a better approach or if my understanding of the method's utility differs from its intended purpose. Your insights are greatly appreciated.

Reply:
Yup, I understand.
My concern was that the flexibility this provides was not actually in use. In general we try not to create things that might be useful in the future but are not actually in use now (YAGNI).